apps/ledger-live-mobile/src/components/CurrencyInput.tsx

```diff
@@ -47,7 +53,10 @@ function format(
 
 type Props = {
   isActive: boolean;
-  onFocus: (_: boolean) => void;
+  onFocus: (
+    _: boolean,
+    event?: NativeSyntheticEvent<TextInputFocusEventData>,
+  ) => void;
```

Comment:
Are the changes in this file mandatory? Can you please explain what issue they solve?
Thanks.